src/main/reactjs/src/tests/cypress/support/utils/option.ts

```diff
@@ -0,0 +1,5 @@
+export const selectOption = (option: string) => {
```

Comment:
Could be renamed to `clickElement` or `clickOptionalElement`. Perhaps also the `option.ts` could be renamed.

Reply:
Jätän vielä tällaiseksi.